webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -197,8 +227,7 @@ class InputForm(SteuerlotseBaseForm):
 
     def _main_handle(self):
         super()._main_handle()
-        self.render_info.back_link_text = _('form.eligibility.marital_status.back_link_text')
-        self.render_info.prev_url = None
+        self.render_info.prev_url = TaxYearEligibilityInputFormSteuerlotseStep.name
```

Comment:
The martial step now has a back button with the tax year question

Reply:
We don't need to manually set the prev_url right? It is handled by the eligibility step chooser right?